### PR TITLE
feat: java.sql.Types convert to openmldb.DataType

### LINKED ISSUE #679: com._4paradigm.openmldb.sdk.Column has a field called sqlType, missing function to convert sqltype to openmldb.datatype
com._4paradigm.openmldb.sdk.Column has a field called sqlType, and provides setter method, maybe i will use `java.sql.Types`。but when i called `com._4paradigm.openmldb.DataType#swigToEnum` method, i get an error。
Because `java.sql.Types` is not `com._4paradigm.openmldb.DataType`。
i think maybe provides some convert method is better or provide some enum. 
for example:
```
public static DataType toDataType(int sqlType) throws SQLException {
        if (Types.BOOLEAN == sqlType) {
            return DataType.kTypeBool;
        } else if (Types.SMALLINT == sqlType) {
            return DataType.kTypeInt16;
        } else if (Types.INTEGER == sqlType) {
            return DataType.kTypeInt32;
        } else if (Types.BIGINT == sqlType) {
          return DataType.kTypeInt64;
        } else if (Types.FLOAT == sqlType) {
            return DataType.kTypeFloat;
        } else if (Types.DOUBLE == sqlType) {
            return DataType.kTypeDouble;
        } else if (Types.VARCHAR == sqlType) {
            return DataType.kTypeString;
        } else if (Types.DATE == sqlType) {
            return DataType.kTypeDate;
        } else if (Types.TIMESTAMP == sqlType) {
            return DataType.kTypeTimestamp;
        } else {
            throw new SQLException("Unexpected Values: " + sqlType);
        }
    }
```

or enum
```
public enum XXX{
      Bool(0),
      Int(2),
      Long(4),
      String(6),
   ....
}
```

### PR DESCRIPTION
**Pull request by shawn-happy**
java.sql.Types convert to openmldb.DataType #679

### CODE REVIEW

File: java/openmldb-jdbc/src/test/java/com/_4paradigm/openmldb/jdbc/CommonTests.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package com._4paradigm.openmldb.jdbc;
```
Comment (aceforeverd) [inline java/openmldb-jdbc/src/test/java/com/_4paradigm/openmldb/jdbc/CommonTests.java]: add a license header

File: java/openmldb-jdbc/src/test/java/com/_4paradigm/openmldb/jdbc/CommonTests.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import com._4paradigm.openmldb.sdk.Common;
+import java.sql.Types;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CommonTests {
+
+  @Test
+  public void testConvertToDataType() throws Exception{
+    DataType stringType = Common.sqlTypeToDataType(Types.VARCHAR);
```
Comment (aceforeverd) [inline java/openmldb-jdbc/src/test/java/com/_4paradigm/openmldb/jdbc/CommonTests.java]: indent=4 space

**Comment by vagetablechicken** [comment]
pr title should not begin with a capital letter,

File: java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/sdk/Common.java [hunk 3]
```diff
@@ -63,4 +63,28 @@ public static int type2SqlType(DataType dataType) throws SQLException {
         }
         return new com._4paradigm.openmldb.sdk.Schema(columnList);
     }
+
+    public static DataType sqlTypeToDataType(int sqlType) throws SQLException {
+        if (Types.BOOLEAN == sqlType) {
```
Comment (aceforeverd) [inline java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/sdk/Common.java]: use switch statement

File: java/openmldb-jdbc/src/test/java/com/_4paradigm/openmldb/jdbc/CommonTests.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+import com._4paradigm.openmldb.DataType;
+import com._4paradigm.openmldb.sdk.Common;
+import java.sql.Types;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CommonTests {
+
+  @Test
+  public void testConvertToDataType() throws Exception{
```
Comment (aceforeverd) [inline java/openmldb-jdbc/src/test/java/com/_4paradigm/openmldb/jdbc/CommonTests.java]: ut failed: https://github.com/4paradigm/OpenMLDB/runs/4210586359?check_suite_focus=true#step:5:381

**Review by vagetablechicken — APPROVED** [review]
LGTM

**Review by aceforeverd — APPROVED** [review]

**Pull request merged**